### ISSUE TITLE
Messaging - In Chat - Multiple Mentions in following line causes UI issue 

### ISSUE THREAD

**Issue by J-Son89** [status-im/status-react#16108]
In the chat window if a user sends a message that has a mention and then sends a following message with another mention that is aligned with the previous mention - these two mentions will overlap.

 See the error here:
![Movie_Recording_2023-05-30_16-48-16](https://github.com/status-im/status-mobile/assets/22799766/9ec5650d-df5c-45fd-b77b-c2f7fbc6d78f)

**Expected result:**

<img width="794" alt="Screenshot 2023-07-06 at 10 20 20" src="https://github.com/status-im/status-mobile/assets/67952253/3a050442-1c56-41c6-8aa6-3211458d5387">

**Comment by ibrkhalil** [comment]
I couldn't reproduce this @J-Son89, Maybe it was fixed by @rahulpsingh here https://github.com/status-im/status-mobile/pull/15723?
Can you please check again and confirm?
![image](https://github.com/status-im/status-mobile/assets/33176106/06a5a0c5-d4d7-4f44-9fb6-189155db6e2d)

**Comment by J-Son89** [comment]
@ibrkhalil  #15723 was merged in April, this issue was created May 31st so I think not.
Although I do believe that the correct fix for this is to correctly set the spacing for the messages which was probably a separate issue and could have been since addressed.
 
cc @qoqobolo & @churik. Can you confirm this is fixed now or should it be worked on?

**Comment by qoqobolo** [comment]
Hey @J-Son89 @ibrkhalil, this is how it looks for me on the latest nightly, so yeah it doesn't seem to be fixed yet

Pixel 7a
![photo_2023-07-05_18-57-53](https://github.com/status-im/status-mobile/assets/67952253/0fe2e3f1-7275-4c8b-9e09-ac01173671ac)

iPhone SE
![photo_2023-07-05_18-57-48](https://github.com/status-im/status-mobile/assets/67952253/dd0f8c58-86f1-433d-a9fe-f11f23ec602c)

I'm also not sure if we have a design for this case. @Francesca-G could you take a look and tell us what the correct spacing should be here?
Thanks!

**Comment by Francesca-G** [comment]
> Hey @J-Son89 @ibrkhalil, this is how it looks for me on the latest nightly, so yeah it doesn't seem to be fixed yet
> 
> Pixel 7a ![photo_2023-07-05_18-57-53](https://user-images.githubusercontent.com/67952253/251224750-0fe2e3f1-7275-4c8b-9e09-ac01173671ac.jpg)
> 
> iPhone SE ![photo_2023-07-05_18-57-48](https://user-images.githubusercontent.com/67952253/251224756-dd0f8c58-86f1-433d-a9fe-f11f23ec602c.jpg)
> 
> I'm also not sure if we have a design for this case. @Francesca-G could you take a look and tell us what the correct spacing should be here? Thanks!

Here's the design for a 2 row message with multiple mentions, they shouldn't be touching as you can see. 
Hope this helps :)

<img width="865" alt="Screenshot 2023-07-06 alle 10 13 02" src="https://github.com/status-im/status-mobile/assets/122543089/45099133-c137-4623-8b11-d309c9a080cc">

**Comment by qoqobolo** [comment]
@Francesca-G thank you very much! Could you also share the figma link with this case, please?
Thanks!

**Comment by Francesca-G** [comment]
> @Francesca-G thank you very much! Could you also share the figma link with this case, please?
> Thanks!

Sure, sorry! Here's the [Figma frame](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=3276%3A206103&mode=design&t=jLboFBfZkiwKyfDn-1)
I will make sure to always include the Figma link from now on!

**Comment by qoqobolo** [comment]
@Francesca-G ah no worries, everything is fine 🙌 Asking now, because for me it's still challenging sometimes to find something in Figma😅 and I was wondering where this specific screen could be located.
Thanks again for your help and quick responses!

**Comment by VladimrLitvinenko** [comment]
Blocked by https://github.com/status-im/status-mobile/issues/16520
